cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/Partitioning.java

```diff
@@ -88,6 +88,11 @@ public FieldType getFieldType(String fieldName) {
     return fields;
   }
 
+  @Override
+  public String toString() {
+    return fields.toString();
```

Comment:
Not sure if I want that. It is used in this log message (PartitionedFieSetDefinition.reconfigure()):

```
        "Partitioning cannot be changed. Existing: %s, new: %s", oldPartitioning, newPartitioning));
```
In this message, having "Partitioning:" twice would be more distracting than helpful.